doc/release-notes-29496.md

```diff
@@ -0,0 +1,10 @@
+Mempool Policy Changes
+----------------------
+
+- Transactions with version number set to 3 are now treated as standard on all networks (#29496),
+  subject to Opt-in Topologically Restricted Until Confirmation (TRUC) Transactions policy as
+  described in BIP 431.  The policy includes limits on spending unconfirmed outputs (#28948), eviction
```

Comment:
could directly link the BIP here: https://github.com/bitcoin/bips/blob/master/bip-0431.mediawiki